Traceback (most recent call last):
  File "/mnt/beegfs/lcesarini/2025_p_irio/src/./clip/clip_ext_wd.py", line 53, in <module>
    df_id_event=pd.read_csv("/mnt/beegfs/lcesarini/2025_p_irio/res/event_id.csv",skiprows=596,header=None)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/luigi.cesarini/.conda/envs/my_xclim_env/lib/python3.11/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/luigi.cesarini/.conda/envs/my_xclim_env/lib/python3.11/site-packages/pandas/util/_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/luigi.cesarini/.conda/envs/my_xclim_env/lib/python3.11/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/luigi.cesarini/.conda/envs/my_xclim_env/lib/python3.11/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/luigi.cesarini/.conda/envs/my_xclim_env/lib/python3.11/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/luigi.cesarini/.conda/envs/my_xclim_env/lib/python3.11/site-packages/pandas/io/parsers/readers.py", line 1735, in _make_engine
    self.handles = get_handle(
                   ^^^^^^^^^^^
  File "/home/luigi.cesarini/.conda/envs/my_xclim_env/lib/python3.11/site-packages/pandas/io/common.py", line 856, in get_handle
    handle = open(
             ^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/mnt/beegfs/lcesarini/2025_p_irio/res/event_id.csv'
srun: error: cn07: task 0: Exited with exit code 1
